charmcraft/commands/store/__init__.py

```diff
@@ -285,3 +285,99 @@ def run(self, parsed_args):
         logger.info(
             "Revision %d of charm %r released to %s",
             revision, charm_name, ", ".join(parsed_args.channels))
+
+
+class StatusCommand(BaseCommand):
+    """List released revisions for a charm."""
+    name = 'status'
+    help_msg = "list released revisions for a charm in the Store"
+
+    def fill_parser(self, parser):
+        """Add own parameters to the general parser."""
+        parser.add_argument('--name', help="the name of the charm to get revisions")
```

Comment:
we shouldn't need to specify what it's for. That is, it _should_ be enough to just say "the name of the charm"; the "for what" should come from context.

However, currently it is not enough, because `--help` does not say what the command _does_. We need to work on that.

Ufa.

Reply:
Yes, we need to revisit all help messages once we add a "long-text help".